test/dummy/db/schema.rb

```diff
@@ -18,9 +18,11 @@
     t.datetime "expires_at", precision: nil, null: false
     t.datetime "claimed_at", precision: nil
     t.string "token_digest", null: false
+    t.string "identifier", null: false
```

Comment:
hmm, with `null: false` it might be worth considering `column "identifier" of relation "passwordless_sessions" contains null values`

Reply:
I think we should go with `null: false` as the _new default_ in new installations but `null: true` in the upgrade instructions? That way it'll handle legacy tables and Rails can still `validates :presence`